.github/workflows/unit-tests.yml

```
@@ -0,0 +1,16 @@
+name: M2 Unit Tests
+on: [push, pull_request]
+
+jobs:
+    unit-tests:
+        name: M2 Unit Tests
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: extdn/github-actions-m2/magento-unit-tests/8.3@master
+              with:
+                composer_name: grin/module
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)